.github/workflows/buildtest.yml

```diff
@@ -62,10 +64,10 @@ jobs:
     - uses: actions/checkout@v3
     
     - name: Install build dependencies
-      run: ${{ matrix.dependencies }}
+      run: ${{matrix.dependencies}}
       
     - name: Configure CMake
-      run: cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{matrix.build-type}} ${{matrix.generator}} ${{matrix.standard}} ${{matrix.cmake-options}}
+      run: cmake -B ${{github.workspace}}/build -DTRIESTE_ENABLE_TESTING=1 -DCMAKE_BUILD_TYPE=${{matrix.build-type}} ${{matrix.generator}} ${{matrix.standard}} ${{matrix.cmake-options}}
```

Comment:
I think you could add "TRIESTE_BUILD_PARSER_TESTS=1" here as well. It might be cleaner to have `${{matrix.testing}}` here and then add that in the matrix above to make it clearer/easier to add testing-specific flags in the future.

Reply:
If it's ok to remove the current matrix of having it both enabled and disabled in different CI runs, sure.

Re: matrix below